---
exceptions: ! "Exception\r\n NoMemoryError\r\n ScriptError\r\n   LoadError\r\n   NotImplementedError\r\n
  \  SyntaxError\r\n SignalException\r\n   Interrupt\r\n     Timeout::Error    # require
  'timeout' for Timeout::Error\r\n StandardError         # caught by rescue if no
  type is specified\r\n   ArgumentError\r\n   IOError\r\n     EOFError\r\n   IndexError\r\n
  \  LocalJumpError\r\n   NameError\r\n     NoMethodError\r\n   RangeError\r\n     FloatDomainError\r\n
  \  RegexpError\r\n   RuntimeError\r\n     Timeout::Error    # moved here in ruby
  1.9.2\r\n   SecurityError\r\n   SocketError\r\n   SystemCallError     # Errno::*
  inherits here\r\n   SystemStackError\r\n   ThreadError\r\n   TypeError\r\n   ZeroDivisionError\r\n
  SystemExit\r\n fatal"
